modules/check_in/app/services/v2/lorota/client.rb

```diff
@@ -78,7 +78,7 @@ def data(token:)
       def connection
         Faraday.new(url: url) do |conn|
           conn.use :breakers
-          conn.response :raise_error, error_prefix: 'LOROTA-MAPPED-API'
+          conn.response :raise_error, error_prefix: 'LOROTA-API'
```

Comment:
looks good, but you can also use the `service_name` variable so it stays in sync with config if we ever change the name